packages/plans-grid-next/src/components/features-grid/client-logo-list/style.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ * Logo list styles
+ */
+
+.plans-grid-next-client-logo-list {
+	display: flex;
+	flex-wrap: wrap;
+	column-gap: 18px;
+	row-gap: 9px;
+
+	svg {
+		display: block;
+		width: auto;
+		height: 24px;
+		fill: var(--studio-gray-50);
+	}
+}
+
+/*
+ * Individual logo adjustments
+ */
+
+.plans-grid-next-client-logo-list__item {
+	--adjustment-unit: calc(24px / 60);
```

Comment:
I noticed these adjustments weren't showing up at all because they lacked a measurement unit (`calc(24 / 60)`), so I fixed it. Maybe it regressed at some point.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/dd433764-6ca8-4f44-8fcc-436db9b81ac7" alt="Logo list, before" width="262">|<img src="https://github.com/user-attachments/assets/4c54cfb0-ea91-4a89-8499-3d599ca07dbb" alt="Logo list, after" width="262">|

<img src="https://github.com/user-attachments/assets/eea9ff3b-549d-47ef-85f9-6e9b58e56fd9" alt="Logo adjustment changes as gif" width="229">